modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -457,6 +457,9 @@ public String toDefaultValue(Property p) {
     @Override
     public String getSwaggerType(Property p) {
         String swaggerType = super.getSwaggerType(p);
+        if (swaggerType.equals("Number")) {
```

Comment:
This is so gross, but that's what happens when class names aren't descriptive. There was a property legitimately named `null` at some point, but I think it got removed.  😅 

I'd add a comment in the source to explain why this silly logic is here so it doesn't get cleaned up accidentally in the future.

Good to merge.